sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -181,13 +183,10 @@ export class EventProcessor {
     stop(): Promise<void>;
 }
 
-// @public (undocumented)
```

Comment:
It looks like you need to rebase on top of what's in master. This section has already been fixed.